demo/src/components/Cart/Cart.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {Container, Col, Row, Form} from 'react-bootstrap';
+import './Cart.css';
+import {setQuantity} from '../../store/StoreHelpers';
```

Comment:
Per the styleguide:
The .js file extension is not optional in import paths and must always be included.

https://google.github.io/styleguide/jsguide.html#es-module-imports

Reply:
Fixed